lib/ffi/library.rb

```diff
@@ -126,7 +126,12 @@ def ffi_lib(*names)
               else
                 # TODO better library lookup logic
                 unless libname.start_with?("/") || FFI::Platform.windows?
-                  path = ['/usr/lib/','/usr/local/lib/','/opt/local/lib/', '/opt/homebrew/lib/'].find do |pth|
+                  path_candidates = if FFI::Platform::ARCH == 'aarch64' && FFI::Platform.mac?
+                    ['/opt/homebrew/lib/','/usr/lib/','/usr/local/lib/','/opt/local/lib/']
+                  else
+                    ['/usr/lib/','/usr/local/lib/','/opt/local/lib/','/opt/homebrew/lib/']
```

Comment:
Could you remove `/opt/homebrew/lib/` from this list? That seems only useful on darwin-arm64.

Also could you add a space after every comma, or even better use `%w[path1 path2]`?